core/runtime/src/main/java/io/quarkus/runtime/graal/Target_javax_management_JMX.java

```diff
@@ -19,4 +21,12 @@ private static <T> T createProxy(MBeanServerConnection connection,
         throw new IllegalStateException("Not Implemented in native mode");
     }
 
+    static final class JmxServerNotIncluded implements BooleanSupplier {
+
+        @Override
+        public boolean getAsBoolean() {
+            String result = System.getProperty("quarkus.native.jmxserver.included");
+            return !"true".equals(result);
```

Comment:
You may also use [`Boolean.html#getBoolean(java.lang.String)`](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/Boolean.html#getBoolean(java.lang.String))

```suggestion
            return Boolean.getBoolean("quarkus.native.jmxserver.included");
```

Reply:
good idea! Done